modules/images/dominant-color/load.php

```diff
@@ -147,7 +147,7 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 	 *
 	 * @param string $content the content to filter.
 	 * @param string $context the context of the content.
-	 * @return string content
+	 * @return string content.
```

Comment:
Documentation should also start with a capital letter. It's also very vague here overall, so potentially worth improving.
```suggestion
	 * @return string The updated $content.
```